games-arcade/rocksndiamonds/rocksndiamonds-3.3.1.2.recipe

```diff
@@ -59,5 +62,9 @@ INSTALL()
 	destdir=$appsDir/"Rocks'n'Diamonds"
 	mkdir -p $destdir
 	cp -r rocksndiamonds sounds graphics levels music $destdir
+
```

Comment:
Hmm, I did not know that _internal_ could be used like this.
That's a good idea!